### DISCUSSION TITLE
Opening mini files on Neovim start

### DISCUSSION THREAD

**Discussion by rpbaptist** [LazyVim/LazyVim#5030] [Q&A]
I have enabled the mini-files plugin and disabled Neotree. When I open Neovim in a directory, I get netrw. When I disable netrw, I get an empty buffer instead.

How can I use mini-files on startup? I followed [the instructions from mini-files](https://github.com/echasnovski/mini.files/blob/main/doc/mini-files.txt#L672-L674), but it's not working for me. I'm thinking it might have to do with LazyVim setup.

[This is my relevant config](https://github.com/rpbaptist/dotfiles/blob/work/config/nvim/lua/plugins/files.lua).

**Comment by dpetka2001** [comment]
I believe you also want 
```lua
options = {
  -- Whether to use for editing directories
  -- Disabled by default in LazyVim because neo-tree is used for that
  use_as_default_explorer = true,
},
```
You might also want to disable `neo-tree` because it will also open at startup.

**Reply by rpbaptist** [reply]
I disabled Neotree here: https://github.com/rpbaptist/dotfiles/blob/work/config/nvim/lua/plugins/core.lua

I don't have `use_as_default_explorer = true`, because it's the default option. I tried setting it anyway and there's no difference unfortunately.

**Reply by dpetka2001** [reply]
LazyVim sets this option by default to `false`, because Neotree is its default file browser. You have to explicitly set it to `true` in your personal configuration.

**Reply by rpbaptist** [reply]
Thank you. I had a look at the LazyVim config and I overlooked that it had to be `opts = { options = {  use_as_default_explorer = true, } }` instead of just `opts = { use_as_default_explorer = true }`
